0025-reverse-nodes-in-k-group/0025-reverse-nodes-in-k-group.py

```diff
@@ -10,11 +10,14 @@ def getTheKth(self, nodes , k):
             k-=1
         return nodes
     def reverseKGroup(self, head: Optional[ListNode], k: int) -> Optional[ListNode]:
+        
+        
         dummy = ListNode()
         dummy.next = head
         prevOnes = dummy
         #NOw dummy and previous one is pointing to the same head 
-    
+        
+        
         #Start catching the end of the group and reversing the group 
         while True:
             endOfGroup = self.getTheKth(prevOnes , k)
```

Comment:
try adding line breaks between the if and while blocks - makes it more readable.

Reply:
Noticed